exchanges/kline/kline.go

```diff
@@ -104,6 +105,34 @@ func CreateKline(trades []order.TradeHistory, interval Interval, p currency.Pair
 	return candles, nil
 }
 
+// SeedFromDatabase returns Item from database seeded data
+func SeedFromDatabase(exchange string, pair currency.Pair, interval Interval, start, end time.Time) (Item, error) {
```

Comment:
Are you planning to expand how the seeding works? Since the seed is only the exchange name, you're not really retrieving seed data here, the candle data is real data, not seeded data. I also think the function name sounds like you're trying to perform a seed, rather than get seed.

Reply:
Yeah wording might not be the best here its not seed to database but from the database for use during backtesting
(e.g its previously synced data that you are pulling out of the database for reuse)